stable/redmine/values.schema.json

```diff
@@ -0,0 +1,185 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+    "redmineUsername": {
+      "type": "string",
+      "title": "Username",
+      "form": true
+    },
+    "redminePassword": {
+      "type": "string",
+      "title": "Password",
+      "form": true,
+      "description": "Defaults to a random 10-character alphanumeric string if not set"
+    },
+    "redmineEmail": {
+      "type": "string",
+      "title": "Admin email",
+      "form": true
+    },
+    "persistence": {
+      "type": "object",
+      "title": "Persistence for Redmine replicas",
+      "form": true,
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "form": true,
+          "title": "Enable persistence",
+          "description": "Enable persistence using Persistent Volume Claims"
+        },
+        "size": {
+          "type": "string",
+          "title": "Persistent Volume Size",
+          "form": true,
+          "render": "slider",
+          "sliderMin": 1,
+          "sliderMax": 100,
+          "sliderUnit": "Gi",
+          "hidden": {
+            "condition": false,
+            "value": "persistence.enabled"
+          }
+        }
+      }
+    },
+    "ingress": {
+      "type": "object",
+      "form": true,
+      "title": "Ingress Details",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "form": true,
+          "title": "Use a custom hostname",
+          "description": "Enable the ingress resource that allows you to access the Redmine installation."
+        },
+        "hostname": {
+          "type": "string",
+          "form": true,
+          "title": "Hostname",
+          "hidden": {
+            "condition": false,
+            "value": "ingress.enabled"
+          }
+        }
+      }
+    },
+    "databaseType": {
+      "type": "string",
+      "form": true,
+      "title": "Database Type",
+      "description": "Allowed values: \"mariadb\" and \"postgresql\""
```

Comment:
you can force the strings using the property `enum`:

```
"enum": ["mariadb", "postgresql"]
```

Also, why don't you add "external"? That way we unblock this schema.

Apart from that, you didn't reply to my question: What would happen if you set databaseType.mariadb and mariadb.enabled=false at the same time? Isn't that an inconsistent status? What I mean, is there a valid use case in which those two parameters may not match?

Reply:
If you set that, you'll face an error (check NOTES.txt)